dist/etc/bash_completion.d/elbencho

```diff
@@ -119,6 +119,11 @@ _elbencho_opts()
         --s3aclverify
         --s3baclget
         --s3baclput
+        --s3statdirs
+        --s3btaginline
+        --s3btaginlineverify
```

Comment:
we need to shorten this (and "s3otaginlineverify") to a max of 16 characters, otherwise the description column in the help output will get too small. i don't really like the ugly abbreviation that will come from this, but "s3otaginlverify" and "s3btaginlverify" would be a possibility, because i had to use "inl" already as an abbreviation for "inline" with another option.